felipe honorio moraes/Super_Hero_Power_Calculator/src/com/fhfelipefh/Main.java

```diff
@@ -0,0 +1,40 @@
+package com.fhfelipefh;
+import java.util.Scanner;
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+        System.out.println("Cálculo da força do Super-Homem");
+        double forca=1;
+        double aux;
+        double aux2;
+        double dist_vis=0;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Insira a quantidade de dias que ele está na terra: ");
+        int dias = scanner.nextInt();
+        if(dias<=365)
+        {
+            System.out.println("Não adquiriu força devido a adaptação!");
+            System.out.println("Força: "+forca+" Humano(s) e a distância que sua visão de calor atinge (cm) "+dist_vis);
+        }
+        else{
+            dias = dias - 365;
+            for(int i = 0; i<dias; ++i){
+                aux = forca*0.03;
+                forca=forca+aux;
+            }
+            if(forca>=10)
+            {
+                dist_vis = 10;
+                for(int j = 0; j<dias; j++)
```

Comment:
Acredito que esse for vai adicionar na visão de calor a mesma quantidade de dias que na forca, quando na verdade eu vou querer apenas os dias após a forca ter chegado a 10, faz sentido?